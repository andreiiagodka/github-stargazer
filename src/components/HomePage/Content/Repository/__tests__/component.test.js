import { shallow } from 'enzyme'

import RepositoryComponent from '..'

describe('<RepositoryComponent />', () => {
  it('renders correctly', () => {
    const props = {
      id: 'id',
      name: 'name',
      stars: 1
    }
    const wrapper = shallow(<RepositoryComponent {...props} />)

    expect(wrapper).toMatchSnapshot()
  })
})
