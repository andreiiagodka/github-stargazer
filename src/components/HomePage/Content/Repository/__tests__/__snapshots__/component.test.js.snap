// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RepositoryComponent /> renders correctly 1`] = `
<Link
  to="/details/id"
>
  <ListGroupItem
    action={true}
    active={false}
    disabled={false}
  >
    <Row
      noGutters={false}
    >
      <Col
        md={7}
      >
        name
      </Col>
      <Col
        className="text-right"
        md={5}
      >
        1
      </Col>
    </Row>
  </ListGroupItem>
</Link>
`;
